base/inference.jl

```diff
@@ -3276,44 +3276,6 @@ function replace_getfield!(ast, e::ANY, tupname, vals, sv, i0)
     end
 end
 
-function code_typed(f, types::ANY; optimize=true)
-    types = to_tuple_type(types)
-    code_typed(call, Tuple{isa(f,Type)?Type{f}:typeof(f), types.parameters...}, optimize=optimize)
```

Comment:
My understanding of #11452 was (and please correct me if I'm wrong). If the module (has to be `baremodule`?) doesn't extend `call` but defined a new `call` in it's own module, `Base.call` will not contain the `call` of that module anymore.